lib/src/values/logic_value.dart

```diff
@@ -724,6 +727,10 @@ abstract class LogicValue implements Comparable<LogicValue> {
   /// radix format.  Space-separation is for ease of reading and is often
   /// in chunks of 4 digits.
   ///
+  /// If the format of then length/radix-encoded string is not completely parsed
+  /// an exception will be thrown.  This can be caused by illegal characters
+  /// in the string or too short or too long of a value string.
```

Comment:
what would be an example of too short?  wouldn't it 0-pad?

Reply:
I'm updating this documentation and fixing the code to make "10'b"  legal.  before it had to have at least one bit.